install.sh

```diff
@@ -60,16 +60,18 @@ echo "CREATE DATABASE ong;" | sudo -u postgres psql
 echo "GRANT ALL PRIVILEGES ON DATABASE ong TO ong;" | sudo -u postgres psql
 
 mkdir -p migrations/auth-server
-unzip -joqq $OSIAM_DIST_DIR/osiam-server/osiam-auth-server/osiam-auth-server.war 'WEB-INF/classes/db/migration/postgresql/*' -d migrations/auth-server
+unzip -joqq $OSIAM_DIST_DIR/auth-server/osiam-auth-server.war 'WEB-INF/classes/db/migration/postgresql/*' -d migrations/auth-server
```

Comment:
No need to change the paths. They stayed the same despite the split of the servers.